### ISSUE TITLE
2025-03-17, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#113]
# Today's Arxiv Papers

(1) [ASMA-Tune: Unlocking LLMs' Assembly Code Comprehension via Structural-Semantic Instruction Tuning](http://arxiv.org/abs/2503.11617v1), 2025-03-14

(2) [Neutralizing Bias in LLM Reasoning using Entailment Graphs](http://arxiv.org/abs/2503.11614v1), 2025-03-14

(3) [Broaden your SCOPE! Efficient Multi-turn Conversation Planning for LLMs using Semantic Space](http://arxiv.org/abs/2503.11586v1), 2025-03-14

(4) [Synthesizing Access Control Policies using Large Language Models](http://arxiv.org/abs/2503.11573v1), 2025-03-14

(5) [Implicit Bias-Like Patterns in Reasoning Models](http://arxiv.org/abs/2503.11572v1), 2025-03-14

(6) [VERIFY: A Benchmark of Visual Explanation and Reasoning for Investigating Multimodal Reasoning Fidelity](http://arxiv.org/abs/2503.11557v1), 2025-03-14

(7) [Similarity-Aware Token Pruning: Your VLM but Faster](http://arxiv.org/abs/2503.11549v1), 2025-03-14

(8) [Potential of large language model-powered nudges for promoting daily water and energy conservation](http://arxiv.org/abs/2503.11531v1), 2025-03-14

(9) [HiTVideo: Hierarchical Tokenizers for Enhancing Text-to-Video Generation with Autoregressive Large Language Models](http://arxiv.org/abs/2503.11513v1), 2025-03-14

(10) [V-STaR: Benchmarking Video-LLMs on Video Spatio-Temporal Reasoning](http://arxiv.org/abs/2503.11495v1), 2025-03-14

(11) [A Review of DeepSeek Models' Key Innovative Techniques](http://arxiv.org/abs/2503.11486v1), 2025-03-14

(12) [Integrating LLMs in Gamified Systems](http://arxiv.org/abs/2503.11458v1), 2025-03-14

(13) [Cerebrum (AIOS SDK): A Platform for Agent Development, Deployment, Distribution, and Discovery](http://arxiv.org/abs/2503.11444v1), 2025-03-14

(14) [D3: Diversity, Difficulty, and Dependability-Aware Data Selection for Sample-Efficient LLM Instruction Tuning](http://arxiv.org/abs/2503.11441v1), 2025-03-14

(15) [Text Compression for Efficient Language Generation](http://arxiv.org/abs/2503.11426v1), 2025-03-14

(16) [Empowering Time Series Analysis with Synthetic Data: A Survey and Outlook in the Era of Foundation Models](http://arxiv.org/abs/2503.11411v1), 2025-03-14

(17) [A Framework for a Capability-driven Evaluation of Scenario Understanding for Multimodal Large Language Models in Autonomous Driving](http://arxiv.org/abs/2503.11400v1), 2025-03-14

(18) [Optimizing Large Language Models for Detecting Symptoms of Comorbid Depression or Anxiety in Chronic Diseases: Insights from Patient Messages](http://arxiv.org/abs/2503.11384v1), 2025-03-14

(19) [Modeling Subjectivity in Cognitive Appraisal with Language Models](http://arxiv.org/abs/2503.11381v1), 2025-03-14

(20) [Annotating Scientific Uncertainty: A comprehensive model using linguistic patterns and comparison with existing approaches](http://arxiv.org/abs/2503.11376v1), 2025-03-14

(21) [Cornstarch: Distributed Multimodal Training Must Be Multimodality-Aware](http://arxiv.org/abs/2503.11367v1), 2025-03-14

(22) [RESPONSE: Benchmarking the Ability of Language Models to Undertake Commonsense Reasoning in Crisis Situation](http://arxiv.org/abs/2503.11348v1), 2025-03-14

(23) [AIstorian lets AI be a historian: A KG-powered multi-agent system for accurate biography generation](http://arxiv.org/abs/2503.11346v1), 2025-03-14

(24) [Rule-Guided Feedback: Enhancing Reasoning by Enforcing Rule Adherence in Large Language Models](http://arxiv.org/abs/2503.11336v1), 2025-03-14

(25) [Safe-VAR: Safe Visual Autoregressive Model for Text-to-Image Generative Watermarking](http://arxiv.org/abs/2503.11324v1), 2025-03-14

(26) [MMS-LLaMA: Efficient LLM-based Audio-Visual Speech Recognition with Minimal Multimodal Speech Tokens](http://arxiv.org/abs/2503.11315v1), 2025-03-14

(27) [Unlocking General Long Chain-of-Thought Reasoning Capabilities of Large Language Models via Representation Engineering](http://arxiv.org/abs/2503.11314v1), 2025-03-14

(28) [Are formal and functional linguistic mechanisms dissociated?](http://arxiv.org/abs/2503.11302v1), 2025-03-14

(29) [GNNs as Predictors of Agentic Workflow Performances](http://arxiv.org/abs/2503.11301v1), 2025-03-14

(30) [BriLLM: Brain-inspired Large Language Model](http://arxiv.org/abs/2503.11299v1), 2025-03-14

(31) [High-Dimensional Interlingual Representations of Large Language Models](http://arxiv.org/abs/2503.11280v1), 2025-03-14

(32) [Line of Duty: Evaluating LLM Self-Knowledge via Consistency in Feasibility Boundaries](http://arxiv.org/abs/2503.11256v1), 2025-03-14

(33) [Reasoning-Grounded Natural Language Explanations for Language Models](http://arxiv.org/abs/2503.11248v1), 2025-03-14

(34) [LLMPerf: GPU Performance Modeling meets Large Language Models](http://arxiv.org/abs/2503.11244v1), 2025-03-14

(35) [Collaboration is all you need: LLM Assisted Safe Code Translation](http://arxiv.org/abs/2503.11237v1), 2025-03-14

(36) [PrivacyScalpel: Enhancing LLM Privacy via Interpretable Feature Intervention with Sparse Autoencoders](http://arxiv.org/abs/2503.11232v1), 2025-03-14

(37) [Exploring the Potential of Large Multimodal Models as Effective Alternatives for Pronunciation Assessment](http://arxiv.org/abs/2503.11229v1), 2025-03-14

(38) [GKG-LLM: A Unified Framework for Generalized Knowledge Graph Construction](http://arxiv.org/abs/2503.11227v1), 2025-03-14

(39) [Can Large Reasoning Models do Analogical Reasoning under Perceptual Uncertainty?](http://arxiv.org/abs/2503.11207v1), 2025-03-14

(40) [LLaVA-MLB: Mitigating and Leveraging Attention Bias for Training-Free Video LLMs](http://arxiv.org/abs/2503.11205v1), 2025-03-14

(41) [Reinforcement Learning Outperforms Supervised Fine-Tuning: A Case Study on Audio Question Answering](http://arxiv.org/abs/2503.11197v1), 2025-03-14

(42) [FastVID: Dynamic Density Pruning for Fast Video Large Language Models](http://arxiv.org/abs/2503.11187v1), 2025-03-14

(43) [Align in Depth: Defending Jailbreak Attacks via Progressive Answer Detoxification](http://arxiv.org/abs/2503.11185v1), 2025-03-14

(44) [Palette of Language Models: A Solver for Controlled Text Generation](http://arxiv.org/abs/2503.11182v1), 2025-03-14

(45) [Towards Extreme Pruning of LLMs with Plug-and-Play Mixed Sparsity](http://arxiv.org/abs/2503.11164v1), 2025-03-14

(46) [Don't Take Things Out of Context: Attention Intervention for Enhancing Chain-of-Thought Reasoning in Large Language Models](http://arxiv.org/abs/2503.11154v1), 2025-03-14

(47) [Don't Forget It! Conditional Sparse Autoencoder Clamping Works for Unlearning](http://arxiv.org/abs/2503.11127v1), 2025-03-14

(48) [Vipera: Towards systematic auditing of generative text-to-image models at scale](http://dx.doi.org/10.1145/3706599.3719757), 2025-03-14

(49) [Limits of KV Cache Compression for Tensor Attention based Autoregressive Transformers](http://arxiv.org/abs/2503.11108v1), 2025-03-14

(50) [Quantifying Interpretability in CLIP Models with Concept Consistency](http://arxiv.org/abs/2503.11103v1), 2025-03-14

(51) [Open3DVQA: A Benchmark for Comprehensive Spatial Reasoning with Multimodal Large Language Model in Open Space](http://arxiv.org/abs/2503.11094v1), 2025-03-14

(52) [OmniDiff: A Comprehensive Benchmark for Fine-grained Image Difference Captioning](http://arxiv.org/abs/2503.11093v1), 2025-03-14

(53) [EmbodiedVSR: Dynamic Scene Graph-Guided Chain-of-Thought Reasoning for Visual Spatial Tasks](http://arxiv.org/abs/2503.11089v1), 2025-03-14

(54) [Prompt Alchemy: Automatic Prompt Refinement for Enhancing Code Generation](http://arxiv.org/abs/2503.11085v1), 2025-03-14

(55) [LLMs are Bug Replicators: An Empirical Study on LLMs' Capability in Completing Bug-prone Code](http://arxiv.org/abs/2503.11082v1), 2025-03-14

(56) [Large Reasoning Models in Agent Scenarios: Exploring the Necessity of Reasoning Capabilities](http://arxiv.org/abs/2503.11074v1), 2025-03-14

(57) [API Agents vs. GUI Agents: Divergence and Convergence](http://arxiv.org/abs/2503.11069v1), 2025-03-14

(58) [DeepSeek Powered Solid Dosage Formulation Design and Development](http://arxiv.org/abs/2503.11068v1), 2025-03-14

(59) [Generative Modelling for Mathematical Discovery](http://arxiv.org/abs/2503.11061v1), 2025-03-14

(60) [BannerAgency: Advertising Banner Design with Multimodal LLM Agents](http://arxiv.org/abs/2503.11060v1), 2025-03-14

(61) [ACMo: Attribute Controllable Motion Generation](http://arxiv.org/abs/2503.11038v1), 2025-03-14

(62) [Beyond A Single AI Cluster: A Survey of Decentralized LLM Training](http://arxiv.org/abs/2503.11023v1), 2025-03-14

(63) [An LLM's Attempts to Adapt to Diverse Software Engineers' Problem-Solving Styles: More Inclusive & Equitable?](http://arxiv.org/abs/2503.11018v1), 2025-03-14

(64) [RONA: Pragmatically Diverse Image Captioning with Coherence Relations](http://arxiv.org/abs/2503.10997v1), 2025-03-14

(65) [TigerLLM -- A Family of Bangla Large Language Models](http://arxiv.org/abs/2503.10995v1), 2025-03-14

(66) [Statistical Impossibility and Possibility of Aligning LLMs with Human Preferences: From Condorcet Paradox to Nash Equilibrium](http://arxiv.org/abs/2503.10990v1), 2025-03-14

(67) [From Dionysius Emerges Apollo -- Learning Patterns and Abstractions from Perceptual Sequences](http://arxiv.org/abs/2503.10973v1), 2025-03-14

(68) [TxAgent: An AI Agent for Therapeutic Reasoning Across a Universe of Tools](http://arxiv.org/abs/2503.10970v1), 2025-03-14

(69) [Combinatorial Optimization for All: Using LLMs to Aid Non-Experts in Improving Optimization Algorithms](http://arxiv.org/abs/2503.10968v1), 2025-03-14

(70) [Empirical Computation](http://arxiv.org/abs/2503.10954v1), 2025-03-13

(71) [Graph-Grounded LLMs: Leveraging Graphical Function Calling to Minimize LLM Hallucinations](http://arxiv.org/abs/2503.10941v1), 2025-03-13

(72) [ChatGPT Encounters Morphing Attack Detection: Zero-Shot MAD with Multi-Modal Large Language Models and General Vision Models](http://arxiv.org/abs/2503.10937v1), 2025-03-13

(73) [OASST-ETC Dataset: Alignment Signals from Eye-tracking Analysis of LLM Responses](http://arxiv.org/abs/2503.10927v1), 2025-03-13

(74) [Learning to Inference Adaptively for Multimodal Large Language Models](http://arxiv.org/abs/2503.10905v1), 2025-03-13

(75) [Taxonomic Reasoning for Rare Arthropods: Combining Dense Image Captioning and RAG for Interpretable Classification](http://arxiv.org/abs/2503.10886v1), 2025-03-13

(76) [Chat-TS: Enhancing Multi-Modal Reasoning Over Time-Series and Natural Language Data](http://arxiv.org/abs/2503.10883v1), 2025-03-13

(77) [SCE: Scalable Consistency Ensembles Make Blackbox Large Language Model Generation More Reliable](http://arxiv.org/abs/2503.10881v1), 2025-03-13

(78) [Teamwork makes the dream work: LLMs-Based Agents for GitHub README.MD Summarization](http://arxiv.org/abs/2503.10876v1), 2025-03-13

(79) [Who Relies More on World Knowledge and Bias for Syntactic Ambiguity Resolution: Humans or LLMs?](http://arxiv.org/abs/2503.10838v1), 2025-03-13

(80) [Thinking Machines: A Survey of LLM based Reasoning Strategies](http://arxiv.org/abs/2503.10814v1), 2025-03-13

(81) [HALURust: Exploiting Hallucinations of Large Language Models to Detect Vulnerabilities in Rust](http://arxiv.org/abs/2503.10793v1), 2025-03-13

(82) [Data Caricatures: On the Representation of African American Language in Pretraining Corpora](http://arxiv.org/abs/2503.10789v1), 2025-03-13

(83) [Vulnerability Detection: From Formal Verification to Large Language Models and Hybrid Approaches: A Comprehensive Overview](http://arxiv.org/abs/2503.10784v1), 2025-03-13

(84) [GoT: Unleashing Reasoning Capability of Multimodal Large Language Model for Visual Generation and Editing](http://arxiv.org/abs/2503.10639v1), 2025-03-13

(85) [HybridVLA: Collaborative Diffusion and Autoregression in a Unified Vision-Language-Action Model](http://arxiv.org/abs/2503.10631v1), 2025-03-13

(86) [UniGoal: Towards Universal Zero-shot Goal-oriented Navigation](http://arxiv.org/abs/2503.10630v1), 2025-03-13

(87) [Siege: Autonomous Multi-Turn Jailbreaking of Large Language Models with Tree Search](http://arxiv.org/abs/2503.10619v1), 2025-03-13

(88) [From TOWER to SPIRE: Adding the Speech Modality to a Text-Only LLM](http://arxiv.org/abs/2503.10620v1), 2025-03-13

(89) [Compositional Subspace Representation Fine-tuning for Adaptive Large Language Models](http://arxiv.org/abs/2503.10617v1), 2025-03-13

(90) [R1-Onevision: Advancing Generalized Multimodal Reasoning through Cross-Modal Formalization](http://arxiv.org/abs/2503.10615v1), 2025-03-13

(91) [CoSTA$\ast$: Cost-Sensitive Toolpath Agent for Multi-turn Image Editing](http://arxiv.org/abs/2503.10613v1), 2025-03-13

(92) [TruthPrInt: Mitigating LVLM Object Hallucination Via Latent Truthful-Guided Pre-Intervention](http://arxiv.org/abs/2503.10602v1), 2025-03-13

(93) [Unlock the Power of Unlabeled Data in Language Driving Model](http://arxiv.org/abs/2503.10586v1), 2025-03-13

(94) [Unveiling the Mathematical Reasoning in DeepSeek Models: A Comparative Study of Large Language Models](http://arxiv.org/abs/2503.10573v1), 2025-03-13

(95) [ASIDE: Architectural Separation of Instructions and Data in Language Models](http://arxiv.org/abs/2503.10566v1), 2025-03-13

(96) [Short-term AI literacy intervention does not reduce over-reliance on incorrect ChatGPT recommendations](http://arxiv.org/abs/2503.10556v1), 2025-03-13

(97) [KUDA: Keypoints to Unify Dynamics Learning and Visual Prompting for Open-Vocabulary Robotic Manipulation](http://arxiv.org/abs/2503.10546v1), 2025-03-13

(98) [PiSA: A Self-Augmented Data Engine and Training Strategy for 3D Understanding with Large Models](http://arxiv.org/abs/2503.10529v1), 2025-03-13

(99) [Probing LLMs for Multilingual Discourse Generalization Through a Unified Label Set](http://arxiv.org/abs/2503.10515v1), 2025-03-13

(100) [Conformal Prediction Sets for Deep Generative Models via Reduction to Conformal Regression](http://arxiv.org/abs/2503.10512v1), 2025-03-13
